addOns/oast/src/main/javahelp/org/zaproxy/addon/oast/resources/help/contents/services/services.html

```diff
@@ -20,8 +20,8 @@ <h1>OAST Services</h1>
 <H3>Default Services</H3>
 
 <ul>
-    <li>BOAST - https://odiss.eu:2096/events</li>
-    <li>Interactsh - https://interactsh.com</li>
+    <li>BOAST - https://odiss.eu:2096/</li>
+    <li>Interactsh - https://app.interactsh.com/</li>
```

Comment:
This no longer matches what the options have.

Reply:
Could be. I was only trying to update the help. If further updates are needed for the services to work that's a separate topic.

> We maintain a list of default Interactsh servers to use with interactsh-client:
> 
>     oast.pro
>     oast.live
>     oast.site
>     oast.online
>     oast.fun
>     oast.me
> 
> Default servers are subject to change/rotate/down at any time, thus we recommend using a self-hosted interactsh server if you are experiencing issues with the default server.

Not they suggest self hosted vs using theirs 🤷‍♂️

Reply:
How is it a separate topic if the help is describing the options?

Reply:
Arguably linking interactsh to the GitHub repo might be best for this location/usage.

Reply:
The Default Services are defined in the options…

Reply:
Okay we can leave the help with useless links.

Reply:
If that was the reason to change the help make them non-links (in any case we need to fix the interacsh one).

Reply:
Make the new values non-links or the old ones?

Reply:
Well I guess the question is what to update it to, and do we bother to try to update people's configs since whatever they have setup token wise is probably invalid for a different host.

Reply:
And whether these service links need to match the options.

Reply:
Should the input for interactsh be an editable combobox with all those defaults they list?

Reply:
Maybe remove this section altogether from this page, and add a `Default Services` section to the individual service pages? So the BOAST one would have `https://odiss.eu:2096/events` and Interactsh one would have the new set of defaults. I would also vote for them to be formatted as code so we're not linking directly to them.

Reply:
IMHO we should not be maintaining lists of defaults (and maybe we should not have any default for interactsh since they can change anytime unlike BOAST).

Reply:
So go back to a text field and have it blank? Or keep the combo and add/default a blank entry?

Reply:
For me the former, we can link to their docs for the user to pick one available/functional.

Reply:
Okay.

I was also thinking the items in the help (ex: the lines these comments are associated with) could link the to docs/repo, which should be more stable.

Reply:
It's probably better to mention in the Interactsh options page that the service URL should be specified by the user. There's also a duplication, since the options page is already linking to the repo.